build/pkgs/anyio/install-requires.txt

```diff
@@ -1 +1 @@
-anyio
+anyio ==4.3.0
```

Comment:
Our `install-requires.txt` files should not pin to specific versions. We do the pinning using the package-version.txt files.
See https://github.com/sagemath/sage/pull/37401

Reply:
I see.  I will update the template in my pypi_spkg package.

Do you have any objections to me posting that package on pypi?

Reply:
Of course, no objections here!

But, I hope it's needless to say, you are of course also very welcome to participate in directly contributing to the `sage_bootstrap` package, where our `sage --package create` script lives.